api/src/test/java/ca/bc/gov/educ/api/distribution/service/DistributionServiceTest.java

```diff
@@ -90,59 +90,59 @@ public class DistributionServiceTest {
 
 	@Test
 	public void testdistributeCredentialsMonthly() {
-		DistributionResponse res = testdistributeCredentials_transcript("MER","USERDIST",false,null);
+		DistributionResponse res = testdistributeCredentials_transcript("MER","MONTHLYDIST",false,null, true);
 		assertNotNull(res);
-		res = testdistributeCredentials_certificate("MER","USERDIST","YED2",null,true);
+		res = testdistributeCredentials_certificate("MER","MONTHLYDIST","YED2",null,true, true);
 		assertNotNull(res);
-		res = testdistributeCredentials_certificate("MER","USERDIST","YEDB",null,false);
+		res = testdistributeCredentials_certificate("MER","MONTHLYDIST","YEDB",null,false, true);
```

Comment:
Getting unit test failure here (line 97)

Reply:
This test is for monthly dist run.  So the activity code, MONTHLYDIST is right.